iwyu_test_util.py

```diff
@@ -92,6 +137,32 @@ def HasTarget(target):
         supported.append(m.group(1))
     return target in supported
 
+  @staticmethod
+  def UsingCXXstdlib(lib):
+    id_macro = {
+      'libcxx' : '_LIBCPP_VERSION',
+      'libstdcxx' : '__GLIBCXX__',
+      'msvc' : '_MSVC_STL_UPDATE', # https://github.com/microsoft/STL/wiki/Macro-_MSVC_STL_UPDATE
+    }
+    if lib not in id_macro:
+      raise SyntaxError('Unknown C++ standard library: %s' % lib)
+
+    return id_macro[lib] in Features._GetCXXPreprocessorMacros()
+
+  @staticmethod
+  def UsingCstdlib(lib):
+    id_macro = {
+      'glibc' : '__GLIBC__',
+      'macos' : '__APPLE_CC__', # This identifies the Apple compiler rather than libc.
+      'msvc' : '_MSC_VER', # This identifies the MS compiler rather than libc.
```

Comment:
It's been a while since I was on Windows, but I think the MS libc is most often refered to as "msvcrt", because that's what the dynamic library housing it is called. The docs refer to it as "Microsoft CRT": https://learn.microsoft.com/en-us/cpp/c-runtime-library/c-run-time-library-reference?view=msvc-170, but I've never seen "mscrt" being used as an acronym for it.

There's also the Universal CRT, "UCRT", which seems to be the successor to msvcrt.

Anyway, both these libraries can be and are used by other compilers than MSVC (e.g. MinGW gcc), so I don't think it's 100% fair to tag them "msvc".

Reply:
I think `msvcrt` would probably be most understood even if it's actually the UCRT. I'll go with that for now

For the C++ library MS seem to just use "STL" or on their Github "MSVC implementation of C++ STL". Are you ok to stick with `msvc` there?

Reply:
Yes, that sounds good for the C++ standard library.